debugview/plugin.cpp

```diff
@@ -195,16 +196,35 @@ class debugview : public threadloop {
                     camera_texture_sizes[1].y(), camera_textures[1]);
         ImGui::End();
 
-        ImGui::SetNextWindowSize(ImVec2(700, 350), ImGuiCond_Once);
-        ImGui::SetNextWindowPos(ImVec2(ImGui::GetIO().DisplaySize.x, ImGui::GetIO().DisplaySize.y), ImGuiCond_Once,
-                                ImVec2(1.0f, 1.0f));
-        ImGui::Begin("Onboard camera views");
-        auto windowSize     = ImGui::GetWindowSize();
-        auto verticalOffset = ImGui::GetCursorPos().y;
-        ImGui::Image((void*) (intptr_t) camera_textures[0], ImVec2(windowSize.x / 2, windowSize.y - verticalOffset * 2));
-        ImGui::SameLine();
-        ImGui::Image((void*) (intptr_t) camera_textures[1], ImVec2(windowSize.x / 2, windowSize.y - verticalOffset * 2));
-        ImGui::End();
+        if (use_cam) {
+            ImGui::SetNextWindowSize(ImVec2(700, 350), ImGuiCond_Once);
+            ImGui::SetNextWindowPos(ImVec2(ImGui::GetIO().DisplaySize.x, ImGui::GetIO().DisplaySize.y), ImGuiCond_Once,
```

Comment:
It seems that there are a lot of redundant code between use_cam and use_rgbd

Reply:
use cam is mostly for stereo while use_rgb is for RGBD. I made these based on @Boyuan-Tian's comments on what if there are multiple streams of images (say both EuRoC and OpenNI). Those code would handle that.

Reply:
I think you are missing my point. I was thinking about encapsulate settings into variables, something like use_cam -> display window (camera_texture) and use_rgbd->displaywindow(rgbd_texture)